README.md

```diff
@@ -104,6 +104,66 @@ const [templateFile, setTemplateFile] = useState<File | null>(null);
   setTemplateFile(e!.target!.files!.item(0));
 }}/>
 ```
+### API
+`connected-workbook` exposes __WorkbookManager__, which generates a workbook with a single loaded query or table data.
+
+
+#### async `generateSingleQueryWorkbook`: `Promise<Blob>`
+
+|param   | type   | required   | description   |
+|---      |---    |---          |---            |
+|query   | [QueryInfo](#queryinfo)   | __required__  | Power Query mashup  | 
```

Comment:
links don't work for me, at least during review, if you are sure they work, great.

Reply:
I think the links work, you just can't use them when in preview mode.